### ISSUE TITLE
Cannot remove file from right click context menu

### ISSUE THREAD

**Issue by qarmin** [strawberrymusicplayer/strawberry#889]
**Describe the bug**
After enabling in settings checkbox to be able to remove music files by clicking at them with right mouse button, I found that this option did not appear in context menu(I can't do screenshot of this because Gnome-Shell not allow to do that).

I tried to restart Strawberry several times, but still always this option is not visible.

**Screenshots:**
I enabled option here:
![Screenshot from 2022-02-05 14-49-50](https://user-images.githubusercontent.com/41945903/152644778-b4ab0db0-eb0b-4b46-9956-ea60f43940ac.png)

**System Information:**
 - Operating system: Ubuntu 22.10
 - Strawberry Version: 1.0.1 and also unstable from PPA

**Comment by jonaski** [comment]
It works here but I see that the delete option in the collection does not work if a cue song is selected. Is it cue songs?

**Comment by jonaski** [comment]
Fixed so deleting cue songs works: https://github.com/strawberrymusicplayer/strawberry/commit/53d519247798bed1c0cc5f1cb83cafda335257fb

**Comment by qarmin** [comment]
This happens with normal mp3 file - [noop-0.1s.mp3.zip](https://github.com/strawberrymusicplayer/strawberry/files/8009850/noop-0.1s.mp3.zip)

On second computer I don't have such problem, so I don't know what may cause this

**Comment by jonaski** [comment]
That doesn't make much sense to me.
Does the delete option work in the playlist right click menu if you enable it under the playlist settings?

**Comment by qarmin** [comment]
No,
Should steps from below work?

https://user-images.githubusercontent.com/41945903/153297208-aaf726c1-c747-4c1e-ade8-9d0d9133ac53.mp4

**Comment by qarmin** [comment]
I have no idea what theme this is(probably default Gnome theme for QT, but it is possible that I installed random one in OS to get better dark theme)
Now it looks better, but still option to remove music files is missing

https://user-images.githubusercontent.com/41945903/153447609-9ea58fda-51fa-4481-b0e8-adfa6525c7ed.mp4

**Comment by jonaski** [comment]
Very strange. I must be missing something. Does you second computer also have Ubuntu 22.10?

**Comment by qarmin** [comment]
Well, it is exactly same OS (Ubuntu 21.10, Gnome-Shell 40.4), I don't have set this option but still I see delete button:
![Zrzut ekranu z 2022-02-11 08-13-24](https://user-images.githubusercontent.com/41945903/153551050-4eabcddf-662a-4a21-b306-c9a8ae141468.png)

**Comment by jonaski** [comment]
Your screenshot is showing the collection settings while the menu is from the playlist, the setting for adding delete to the playlist is in the playlist settings.

**Comment by qarmin** [comment]
Yeah, it works now, thanks.

But for me it is quite not intuitive to have 2 different options for one setting.

**Comment by jonaski** [comment]
It makes to me to have it that way because enabling the delete from disk in the playlist is more risky for accidentally deleting files than enabling it in the collection. So it should be possible to enable just one of the options. It also makes sense since it's two separate menus and how the settings are divided up.
I started up Ubuntu 21.10 in a virtualbox to test this and I checked that delete from disk shows up both in the collection and in the playlist when checked. I see now that in the video you posted you show the collection setting while using the playlist menu.
